ui/client/components/HistoryItem.tsx

```diff
@@ -0,0 +1,69 @@
+import {css, cx} from "emotion"
+import React, {useMemo} from "react"
+import {useTranslation} from "react-i18next"
+import styles from "../stylesheets/processHistory.styl"
+import Date from "./common/Date"
+import {ReactComponent as Badge} from "./deployed.svg"
+import {ActionType, ProcessVersionType} from "./Process/types"
+
+type HistoryItemProps = {
+  isLatest?: boolean,
```

Comment:
Hmm, as I remeber each object has isLatest and isDeploted props so you can remove option `?`

Reply:
what you are talking about?